components/Tabs/src/MDCTabBar.m

```diff
@@ -312,6 +312,7 @@ + (MDCItemBarStyle *)defaultStyleForPosition:(UIBarPosition)position
     style.titleFont = [MDCTypography buttonFont];
     style.inkStyle = MDCInkStyleBounded;
     style.titleImagePadding = (kImageTitleSpecPadding + kImageTitlePaddingAdjustment);
+    style.titleNumberOfLines = 2;
```

Comment:
Why are we choosing whether to wrap base on if the TabBar is top or bottom?
(I couldn't find guidance in the spec)

Reply:
The Material "Tabs" component is only at the top of a view. The other component is "[Bottom Navigation](https://material.io/guidelines/components/bottom-navigation.html)" and I do not believe they support more than 1 line of text.